### PR TITLE
fix(gitlab): fix confusing tables

### LINKED ISSUE #1271: confusing gitlab tables
## Describe the bug
confusing tables

## Expected behavior
combine gitlab_commits and gitlab_merge_request_commits
change gitlab_merge_request_commit_merge_requests to gitlab_merge_request_commits

## Screenshots
image

### PR DESCRIPTION
**Pull request by warren830**
# Summary

combine gitlab_commits and gitlab_merge_request_commits
change gitlab_merge_request_commit_merge_requests to gitlab_merge_request_commits

### Key Points

- [ ] This is a breaking change
- [ ] New or existing documentation is updated

### Description
Describe what this PR does, and aims to solve in a few sentences.

### Does this close any open issues?
Close #1271

### Current Behavior
Describe the current behaviour of this issue, if relevant.

### New Behavior
Describe the new behaviour updated in this issue, if relevant.

### Screenshots
image
gitlab_merge_request_commits
image
gitlab_commits
image
gitlab_merge_request_notes
image
gitlab_merge_requests: first_comment_time get the right value 
image
image

### Other Information
Any other information that is important to this PR.

### CODE REVIEW

**Comment by klesh** [comment]
@warren830 
nice, can we do sth about the column name as well?
like PK in `gitlab_commits` is named `sha` , but named `merge_request_commit_id` in `gitlab_merge_request_commits`

File: plugins/gitlab/tasks/gitlab_merge_request_commits_cpllector.go [hunk 1]
```diff
@@ -69,9 +69,9 @@ func CollectMergeRequestCommits(projectId int, mr *models.GitlabMergeRequest, gi
 		})
 }
 
-func convertMergeRequestCommit(commit *GitlabMergeRequestCommit) (*models.GitlabMergeRequestCommit, error) {
```
Comment (klesh) [inline plugins/gitlab/tasks/gitlab_merge_request_commits_cpllector.go:72]: I think this task should be deleted, right? we have a task to convert GitlabCommit already.

File: plugins/gitlab/tasks/gitlab_merge_request_commits_cpllector.go [hunk 2]
```diff
@@ -41,20 +41,20 @@ func CollectMergeRequestCommits(projectId int, mr *models.GitlabMergeRequest, gi
 				return nil
 			}
 			for _, commit := range *gitlabApiResponse {
-				gitlabMergeRequestCommit, err := convertMergeRequestCommit(&commit)
+				gitlabCommit, err := convertMergeCommitToCommit(&commit)
```
Comment (klesh) [inline plugins/gitlab/tasks/gitlab_merge_request_commits_cpllector.go:44]: the file name has a typo

Comment (warren830) [reply plugins/gitlab/tasks/gitlab_merge_request_commits_cpllector.go]: fixed

File: plugins/gitlab/tasks/gitlab_merge_request_commits_cpllector.go (hunk 1 above)
Comment (warren830) [reply plugins/gitlab/tasks/gitlab_merge_request_commits_cpllector.go]: fixed

**Comment by warren830** [comment]
> 



> @warren830 nice, can we do sth about the column name as well? like PK in `gitlab_commits` is named `sha` , but named `merge_request_commit_id` in `gitlab_merge_request_commits`

fixed

**Review by klesh — APPROVED** [review]
LGTM

**Pull request merged**